### PR TITLE
Implement Desktop version of contact form and footer section

### PR DESCRIPTION
**Pull request by desorgui**
## What?
Implementation of the desktop version of the contact form and footer

## Why?
These changes complete the project requirements.

## How?
I used Html and CSS (Grid and Flexbox) to be able to place each element of my project, and also change the display for mobile
## Testing?
I've added Linter for testing all new CSS codes.

## Anything Else?
I used [template 2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) of the Microverse students' projects to implement my portfolio

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -22,6 +22,28 @@ button {
   cursor: pointer;
 }
```
Comment (gemmen29) [inline style.css:24]: - [ ] Kindly use a min-width media query for the desktop version from any point between `768px` to `1440px` as this project is the recommended width for the desktop version and this project is mobile-first.

for example:
```
@media screen and (min-width: 1080px) { // any value between 768 and 1440 is ok.
/*
.
.
*/
}

```

File: index.html [hunk 2]
```diff
@@ -72,33 +75,33 @@ <h4>I'm a software Developer</h4>
 			</div>
 		</section>
 		<section class="my-work" id="my-work">
-			<h1 class="section-title">My Recents works</h1>
+			<h1 class="section-title font-crete">My Recents works</h1>
```
Comment (gemmen29) [inline index.html]: - [ ] Kindly add some spacing on the left and right in the `My Recents works` section `mobile-version` to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image |image |

File: index.html [hunk 3]
```diff
@@ -194,48 +197,52 @@ <h3 class="work-title">My Work Title</h3>
...
-					<button class="see-project-btn">See Project</button>
+					<button class="see-project-btn font-inter">See Project</button>
 				</div>
 			</div>
 		</section>
 		<section class="about-me" id="about-me">
 			<div class="resume">
-				<h1>About me</h1>
-				<p class="resume-text">
+				<h1 class="font-crete">About me</h1>
```
Comment (gemmen29) [inline index.html:207]: - [ ] Kindly add some spacing between the `about me` section and the white horizontal line in `desktop-version` to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @desorgui, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -369,176 +561,476 @@ header .nav-menu ul li a:hover {
...
+  margin-right: 20px;
+}
+
+/* ============================================
+==============================================
+==============================================
+============================================== */
+
+@media screen and (min-width: 1080px) {
   body {
```
Comment (gtekle) [inline style.css]: - [ ] You should use `min-width: 768px;` as the previous reviewer suggested and as per the project requirement.
desktop-vers-768-instruct

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @desorgui :wave: , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
dog-pug

 ## Highlights
:heavy_check_mark: Professional Readme
:heavy_check_mark: Descriptive pull request summary

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ Optional ] Kindly update the left alignment of the paragraph in the contact-me section as in the figma design. 
contact-me-parag-left-align-impl

- [ Optional ] Kindly left align (Flex start) on the Desktop version for the Skills section.
#### Figma
skills-section-left-alignment-figma
#### Impl
skills-section-left-alignment-impl


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey  @desorgui 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

#### Optional
Please remove the extra spaces from the sides.
Screen Shot 2022-05-08 at 7 43 26 AM


# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**